draco/synthesis/noise.py

```diff
@@ -53,6 +53,67 @@ def process(self, data):
         return data
 
 
+class GaussianNoise(task.SingleTask):
```

Comment:
Please document the parameters below in an `Attributes` section to this doc string.